lib/chrome.js

```diff
@@ -58,6 +59,16 @@ class Events {
 
 const MIN_INTERVAL = 1000;
 
+function keyToId(key) {
+  let md5Str = md5(key);
+  const parts = [];
+  [8,4,4,4,12].map(numChars => {
```

Comment:
I remember it was possible to pass a 16 bytes array to the v4 constructor random parameter.